### PR TITLE
improvements to docs layout and index page

### PR DESCRIPTION
**Pull request by czepluch**

### CODE REVIEW

File: docs/overview_and_guide.rst [hunk 1]
```diff
@@ -216,4 +220,4 @@ In order to use Raiden with an rpc-endpoint provided by an Infura Ethereum node,
 Select the desired Ethereum account when prompted, and type in the account's password.
 
 
-See the :doc:`API walkthrough <api_walkthrough>` for further instructions on how to interact with Raiden.
+Now that Raiden is up and running, head over to the :doc:`API walkthrough <api_walkthrough>` for further instructions on how to interact with Raiden. There's also a :doc:`Web UI tutorial <webui_tutorial>` available for people who prefer a graphical approach.
```
Comment (palango) [inline docs/overview_and_guide.rst]: maybe *interface* instead of *approach*?

File: docs/_static/css/custom.css [hunk 2]
```diff
@@ -0,0 +1,6 @@
+@import url("default.css");
+
+/*Make sidebar captions steelblue*/
+span.caption-text {
+  color: steelblue;
```
Comment (palango) [inline docs/_static/css/custom.css]: Do you have a screenshot on how this looks?

File: docs/index.rst [hunk 3]
```diff
@@ -8,9 +8,39 @@ What is the Raiden Network?
...
+New to Raiden?
+~~~~~~~~~~~~~~
+If you're new to Raiden and just want to install and try it out:
+
+    * :doc:`Install and fire up <overview_and_guide>` a Raiden client.
+    * Check out the :doc:`WebUI tutorial <webui_tutorial>` to quickly get started doing payments.
+
+Developing on Raiden?
+~~~~~~~~~~~~~~~~~~~~~
+If you're looking for documentation relevant for building on to of Raiden:
```
Comment (palango) [inline docs/index.rst]: s/on to/on top

File: docs/index.rst [hunk 4]
```diff
@@ -8,9 +8,39 @@ What is the Raiden Network?
...
+
+    * Read the thorough guide to :doc:`get started with the Raiden API <api_walkthrough>`.
+    * Checkout the complete :doc:`API documentation <rest_api>`.
+
+Contributing to Raiden?
+~~~~~~~~~~~~~~~~~~~~~~~
+If you're looking for a more in depth understanding of Raiden and is interested in contributing:
```
Comment (palango) [inline docs/index.rst]: s/and is/ and are

File: docs/index.rst [hunk 5]
```diff
@@ -8,9 +8,39 @@ What is the Raiden Network?
...
+    * Take a look at the :doc:`onboarding guide <onboarding>`.
+    * Check out the code in our `Github repository <https://github.com/raiden-network/raiden>`_.
+    * Read the `Raiden specification <https://raiden-network-specification.readthedocs.io/en/latest/index.html>`_.
+
+Need help?
+~~~~~~~~~~
+If you run in to any kind of problems or you discover bugs:
+
+    * Ask for help or project related questions in the `Raiden gitter <https://gitter.im/raiden-network/raiden>`_.
+    * Create a `Github issue <https://github.com/raiden-network/raiden/issues/new/choose>`_ if you discover a bug or has a feature request.
```
Comment (palango) [inline docs/index.rst]: - s/discover/dicovered
- s/has/have

File: docs/_static/css/custom.css (hunk 2 above)
Comment (czepluch) [reply docs/_static/css/custom.css]: 2019-03-15-121049_654x1044_scrot
Don't mind the weird line, that's a problem with my screenshot program.

File: docs/index.rst (hunk 5 above)
Comment (czepluch) [reply docs/index.rst]: I think it's fine to keep `discover`.

File: docs/_static/css/custom.css (hunk 2 above)
Comment (palango) [reply docs/_static/css/custom.css]: Looks nice!

**Pull request merged**